lstore/table.py

```diff
@@ -401,5 +399,12 @@ def write_record(self, record: Record, rid: int) -> bool:
         return True
 
 
-    def read_record(self) -> bool:
-        pass
+    def read_record(self, key):
+        record = Record(key, 0, "00000", [1, 2, 3, 4, 5, 6])
+        return record
+
+    def record_does_exist(self, key) -> bool:
+        return True
+
+    def new_tid(key) -> int:
+        return 1
```

Comment:
I can implement this when I change the indirection column logic if that is what we want to do.